web/src/components/apps/AppConfig.jsx

```diff
@@ -414,7 +414,7 @@ class AppConfig extends Component {
 
   isConfigReadOnly = (app) => {
     const { match } = this.props;
-    if (!match.params.sequence) return false;
+    if (!match.params.sequence) {return false;}
```

Comment:
was this generated by some prettier?

Reply:
Definitely generated.  I think it was ESLint